src/evops/utils/metrics_utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022, Skolkovo Institute of Science and Technology (Skoltech)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Any
+from nptyping import NDArray
+
+import numpy as np
+
+
+def __group_indices_by_labels(
+    labels_array: NDArray[Any, np.int32],
+) -> Dict[np.int32, NDArray[Any, np.int32]]:
+    """
+    :param labels_array: list of point cloud labels
+    :return: dictionary with labels and an array of indices belonging to this label
+    """
+    unique_labels = np.unique(labels_array)
```

Comment:
`return_index` property of `np.unique` is doing the same thing for you, without `np.where`